python3-sys/build.rs

```diff
@@ -388,6 +389,10 @@ fn main() {
     // If you have troubles with your shell accepting '.' in a var name, 
     // try using 'env' (sorry but this isn't our fault - it just has to 
     // match the pkg-config package name, which is going to have a . in it).
+    match python_sys::python_path() {
+        Some(path) => env::set_var("PYTHON_SYS_EXECUTABLE", path),
```

Comment:
This is a ugly way to hook into the build-time configuration of the python3-sys library. The preferable way to do this would be to refactor how we use [`fine_interpreter_and_get_config`](https://github.com/dgrunwald/rust-cpython/pull/131/files#diff-646601a8a4d8d5c828181358c7c201ceL254)-- only calling that method if we searching for a python install. Presumably if we're building and supplying our own python, then we should also be able to take our build time configuration and patch through those variables supplied by this build script (e.g. `Py_USING_UNICODE`).